packages/babel-register/package.json

```diff
@@ -17,7 +17,8 @@
   "dependencies": {
     "@cspotcode/source-map-support": "condition:BABEL_8_BREAKING ? ^0.8.1 :",
     "clone-deep": "^4.0.1",
-    "find-cache-dir": "condition:BABEL_8_BREAKING ? ^3.3.1 : ^2.0.0",
+    "find-cache-dir": "condition:BABEL_8_BREAKING ? ^5.0.0 : ^2.0.0 (esm:default)",
+    "lmdb": "condition:BABEL_8_BREAKING ? ^3.2.6 : (esm:open)",
```

Comment:
Can we make LMDB an optional dependency? The node-lmdb depends on node-gyp-build, which will kick it when the prebuilt binary bindings does not match the user arch. It is likely to happen when we still support a node version that is not supported by the prebuilt.

This opens all sort of install issues because of the tools/libs required and users may not be familiar with them.

See also https://github.com/parcel-bundler/parcel/issues/8152 for an example.

If we make it an optional dependency, users can still use Babel register regardless of node-gyp-errors, though the cache handling is not as efficient as it would be.

Reply:
I agree that this is an annoying problem.
But I worry that making it an optional dependency will make almost no one use it.

Reply:
I investigated and the pre-built packages released by lmdb support node-api, so the version shouldn't be a problem.
But they do not provide some pre-built packages compared to rollup, such as the lack of Windows x86 and arm.